kaldb/src/test/java/com/slack/kaldb/testlib/ChunkManagerUtil.java

```diff
@@ -74,6 +108,11 @@ public void close() throws IOException, TimeoutException {
     if (s3Client != null) {
       s3Client.close();
     }
+    metadataStoreService.stopAsync();
+    metadataStoreService.awaitTerminated(DEFAULT_START_STOP_DURATION);
+    if (localZkServer != null) {
```

Comment:
Shouldn't need the null check here either?

Reply:
We employ null checks here since some unit tests kill terminate the ZK server and set this variable to null. It is a convention I adopted for tests, so the teardown code can know this to be the case.